dissect/cstruct/exceptions.py

```diff
@@ -12,3 +12,8 @@ class ResolveError(Error):
 
 class NullPointerDereference(Error):
     pass
+
+
+class ArraySizeMismatch(Error):
```

Comment:
```suggestion
class ArraySizeError(Error):
```
I've been trying to think if there was some Python builtin error we could use but came up blank. I think it's better to rename this to be more consistent with other error names though.